AlertaDengue/dados/views.py

```diff
@@ -564,6 +565,26 @@ def get_context_data(self, **kwargs):
             total_series = [0]
             total_observed_series = [0]
 
+        menssagem = _(
```

Comment:
as you are using this variable for `except ValueError`, move it to `except ValueError:` scope

Reply:
Usando o gettex() com a sintaxe "f" o django estava gerando um erro nas multilinhas para traduzir, voltei ao modo format()...